doc/libnvme.rst

```diff
@@ -6340,12 +6339,12 @@ Returns true if given offset is 64bit register, otherwise it returns false.
   NVM subsystem over a 10 second period in this power state with
   the workload indicated in the Active Power Workload field.
 
-``apw``
+``apws``
+  APW occupy the first 3 bits of the field
```

Comment:
Could you use something like 'Bits 2-0:' to be more precises in the wording? This aligns also to the existing bit field description.

Reply:
Thanks for the comments, I Just modified that explanation